ironic_prometheus_exporter/tests/test_ipmi_parser.py

```diff
@@ -80,15 +106,27 @@ def test_system_parser(self):
         self.assertIn('baremetal_system_post_err', system_metrics_name)
 
         ipmi.prometheus_format(self.payload['System'], self.node_name,
+                               self.node_uuid, self.instance_uuid,
+                               self.timestamp,
```

Comment:
you can remove one line if you add the next variable on this line instead of splitting